docs/content/Configuration/Config.md

```diff
@@ -481,6 +490,43 @@ module.exports = {
 CORS settings for the Cube.js REST API can be configured by providing an object
 with options [from here][link-express-cors-opts].
 
+### jwt
+
+#### jwkUrl
+
+The URL from which JSON Web Key Sets (JWKS) can be retrieved. Can also be set
+using `CUBEJS_JWK_URL`.
+
+#### key
+
+A JSON string that represents a cryptographic key and its' properties. Can also
+be set using `CUBEJS_JWK_KEY`.
+
+#### algorithms
+
+[Any supported algorithm for decoding JWTs][gh-jsonwebtoken-algs]. Can also be
```

Comment:
Is it really any? @ovr

Reply:
Digging through the source code, it might be limited to the following:

```
'RS256', 'RS384', 'RS512', 'ES256', 'ES384', 'ES512', 'HS256', 'HS384', 'HS512'
```

Can you confirm, @ovr?

Reply:
Yeah, it's limited to ES/RS/HS.